Tests/Data/Elliptic/square_100x100_ComponentTransport/square_1e4_heterogeneity.prj

```diff
@@ -37,10 +37,19 @@
                     </storage>
                 </porous_medium>
             </porous_medium>
+            <components>
+                <component name="Si">
+                    <diffusion>
+                        <type>Constant</type>
+                        <diffusion_parameter>Dm</diffusion_parameter>
+                    </diffusion>
+                    <dispersivity>
+                        <type>Constant</type>
+                        <dispersivity_parameters>beta</dispersivity_parameters>
```

Comment:
I like the previous description of the parameters using the tags `solute_dispersivity_longitudinal` and `solute_dispersivity_transverse` more since the user should be more familiar with these terms when he studied the accompanied documentation.

Reply:
OK. I will revert it back to the previous status.